mod-notes/mod-notes.postman_collection.json

```diff
@@ -8214,6 +8214,306 @@
 				}
 			]
 		},
+		{
+			"name": "limit config",
+			"item": [
+				{
+					"name": "/configurations/entries - POST note types limit to 3",
```

Comment:
Prior to updating configuration setting to 3 -- could we check if the configuration had previously been set and preserve the setting? We follow a similar process in mod-kb-ebsco-java api test when setting api key (for example) -- since we don't want to alter an environment in which the test gets run

Reply:
Thanks @carolegodfrey for the comment. I have added check for configuration. Now when we have existing configuration it will be updated to our setting(value 3 - needed for tests). And after all tests it will be back to old one.

Reply:
Great -- thank you @stanislav-mureiko